bus/client.go

```diff
@@ -497,13 +497,14 @@ func (c *Client) SearchObjects(ctx context.Context, key string, offset, limit in
 // Object returns the object at the given path with the given prefix, or, if
 // path ends in '/', the entries under that path.
 func (c *Client) Object(ctx context.Context, path, prefix string, offset, limit int) (o object.Object, entries []api.ObjectMetadata, err error) {
+	path = strings.TrimPrefix(path, "/")
 	values := url.Values{}
 	values.Set("prefix", prefix)
 	values.Set("offset", fmt.Sprint(offset))
 	values.Set("limit", fmt.Sprint(limit))
-	path = strings.TrimLeft(path, "/")
+	endpoint := fmt.Sprintf("%s?%s", path, values.Encode())
 	var or api.ObjectsResponse
-	err = c.c.WithContext(ctx).GET(fmt.Sprintf("/objects/%s?"+values.Encode(), path), &or)
+	err = c.c.WithContext(ctx).GET(fmt.Sprintf("/objects/%s", endpoint), &or)
```

Comment:
Imo the double `fmt.Sprintf` is more confusing than helpful. `fmt.Sprintf("/objects/%s?%s", path, values.Encode()` reads nicer to me.

Reply:
`jape` complains about that, it thinks we're passing two parameters while we only expect one